cypress/e2e/awx/administration/topology-view.cy.ts

```diff
@@ -64,59 +63,53 @@ describe('Topology view', () => {
       });
   });
 
-  it('does not show Topology View in sidebar for non admins', function () {
-    cy.createAwxUser(organization).then((awxUser) => {
-      user = awxUser;
-
-      cy.awxLoginTestUser(user.username, 'pw');
-      cy.getByDataCy('page-navigation').then((nav) => {
-        if (!nav.is(':visible')) cy.getByDataCy('nav-toggle').click();
-      });
-      cy.get('[data-cy="awx-topology-view"]').should('not.exist');
-
-      cy.visit('/infrastructure/topology');
-      cy.contains('Page not found');
-      cy.contains('We could not find that page.');
-    });
-  });
-
-  it('will allow the user to view a large number of nodes', () => {
-    cy.fixture('instance_nodes').then((instanceNodes: MeshVisualizer) => {
-      cy.intercept('GET', awxAPI`/mesh_visualizer/`, instanceNodes);
-
-      cy.navigateTo('awx', 'topology-view');
-
-      instanceNodes.nodes.forEach((node) => {
-        cy.contains(node.hostname);
-      });
-    });
-  });
-
   it('will allow the user to select node and delete it', () => {
     cy.requestGet<Settings>(awxAPI`/settings/system/`).then((data) => {
       if (!data?.IS_K8S) {
         cy.log('TEST SKIPPED | IS_K8S: False');
         return;
       }
       const node = 'E2EInstance' + randomString(4);
-
       cy.createAwxInstance(node).then(() => {
         cy.navigateTo('awx', 'topology-view');
         cy.contains(node).click({ force: true });
         cy.getByDataCy('mesh-viz-sidebar').within(() => {
           cy.getByDataCy('name').contains(node).click();
         });
-
         cy.url().should('include', '/infrastructure/instances/');
         cy.getByDataCy('page-title').contains(node);
-
         cy.clickKebabAction('actions-dropdown', 'remove-instance');
         cy.clickModalConfirmCheckbox();
         cy.clickButton('Remove instance');
-
         cy.navigateTo('awx', 'topology-view');
         cy.contains(node).should('not.exist');
       });
     });
   });
+
+  it('will allow the user to view a large number of nodes', () => {
+    cy.fixture('instance_nodes').then((instanceNodes: MeshVisualizer) => {
+      cy.intercept('GET', awxAPI`/mesh_visualizer/`, instanceNodes);
+      cy.navigateTo('awx', 'topology-view');
+      instanceNodes.nodes.forEach((node) => {
+        cy.contains(node.hostname);
+      });
+    });
+  });
+
+  tag(['upstream'], () => {
```

Comment:
@akus062381 @jamestalton do we need to modify package.json to run those?

Reply:
@nixocio I did modify the package.json in the downstream repo